### PR TITLE
Update to use new WOF domain

### LINKED ISSUE #1488: Switch whosonfirst.mapzen.com to data.whosonfirst.org
There are a few spots where we point to whosonfirst.mapzen.com:

- [ ] integration tests
- [ ] documentation
- [ ] raw tiles
- [ ] attribution.md
- [ ] layers.md
- [ ] ...

Those should be switched to whosonfirst.org.

e.g. https://whosonfirst.mapzen.com/data/x/y/z/xyz.geojson ➡️  https://data.whosonfirst.org/x/y/z/xyz.geojson

### PR DESCRIPTION
**Pull request by iandees**
Fixes #1488.

### CODE REVIEW

File: docs/attribution.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ More details can be found on the Natural Earth [Terms of Use](http://www.natural
 
 `source:whosonfirst`
 
-Crediting [Who's On First](https://whosonfirst.mapzen.com) is recommended and linking back to their License is required.
+Crediting [Who's On First](https://www.whosonfirst.org) is recommended and linking back to their License is required.
 
 ```
-Data from Who's On First. <a href="https://whosonfirst.mapzen.com#License">License</a>
+Data from Who's On First. <a href="https://www.whosonfirst.org">License</a>
```
Comment (nvkelso) [inline docs/attribution.md]: This URL is better to link to: https://www.whosonfirst.org/docs/licenses/.

Though this is the ultimate truth: https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/README.md

File: docs/data-sources.md [hunk 2]
```diff
@@ -45,4 +45,4 @@ More details can be found at: http://www.naturalearthdata.com/about/terms-of-use
 
 `source:whosonfirst`
 
-[Who's On First](http://whosonfirst.mapzen.com) is Mapzen's place gazetteer, and is used to source neighbourhood labels.
+[Who's On First](http://www.whosonfirst.org/) is an open place gazetteer created by Mapzen and is used to source neighbourhood labels.
```
Comment (nvkelso) [inline docs/data-sources.md]: `created` > `initiated`

**Comment by nvkelso** [comment]
We also need to update RAWR Tiles generation:

- https://github.com/tilezen/raw_tiles/blob/c7a042cd64a8b29ca3ab9919893c06bc37ecd3b7/raw_tiles/source/wof_neighbourhood.sql#L8

File: docs/attribution.md (hunk 1 above)
Comment (nvkelso) [reply docs/attribution.md]: Data from Who's On First. <a href="https://www.whosonfirst.org">License</a> and <a href="https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/README.md">Sources</a>

**Comment by iandees** [comment]
From https://github.com/tilezen/raw_tiles/pull/21#issuecomment-382451197; we probably need to update tilequeue as well:

https://github.com/tilezen/tilequeue/blob/36fca4824730cc0eeabb54f53b0f7f15b35423a4/tilequeue/process.py#L243-L244

**Comment by nvkelso** [comment]
With the tilequeue change in I re-ran the CI and it passes! Merging :)

**Comment by iandees** [comment]
Thanks for re-running!

**Pull request merged**